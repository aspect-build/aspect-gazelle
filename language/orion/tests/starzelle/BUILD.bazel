load("@aspect_gazelle//:gazelle.bzl", "gazelle_generation_test")

# General starzelle unit tests. Each test has its own *.star plugins and only the host language.

[
    gazelle_generation_test(
        name = "%s_test" % t,
        dir = t,
        env = {
            "TEST_ORION_EXTENSIONS": "%s/%s" % (
                package_name(),
                t,
            ),
        },
        gazelle_binary = "//tests:gazelle_orion_binary",
    )
    for t in [t.replace("/WORKSPACE", "") for t in glob(["*/WORKSPACE"])]
]
