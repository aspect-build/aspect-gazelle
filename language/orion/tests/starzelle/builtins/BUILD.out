filegroup(
    name = "all-files",
    srcs = [
        "copy-txt.axl",
        "filegroup.axl",
    ],
)
